### PR TITLE
Fwmt 696 code review

### PR DESCRIPTION
**Pull request by sdiaz1000**

### CODE REVIEW

File: src/main/java/uk/gov/ons/fwmt/census/rmadapter/canonical/CanonicalJobHelper.java [hunk 1]
```diff
@@ -38,57 +34,54 @@ public FWMTCreateJobRequest createJob(ActionInstruction actionInstruction) throw
     // TODO not yet implemented in Canonical
     //address.setCategory(actionAddress.getCategory());
 
-    fwmtCreateJobRequest.setJobIdentity(actionRequest.getCaseRef()); 
-    fwmtCreateJobRequest.setSurveyType(actionRequest.getSurveyRef()); 
+    createJobRequest.setJobIdentity(actionRequest.getCaseRef());
```
Comment (garethTurner) [inline src/main/java/uk/gov/ons/fwmt/census/rmadapter/canonical/CanonicalJobHelper.java:37]: this should be caseId and case ref needs added to create job request. Both are always required so neither should be an additional property

File: src/main/java/uk/gov/ons/fwmt/census/rmadapter/config/GatewayActionsQueueConfig.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  @Autowired
+  private AmqpAdmin amqpAdmin;
+  
+  public static final String GATEWAY_ACTIONS_QUEUE = "Gateway.Actions";
+  public static final String GATEWAY_ACTIONS_EXCHANGE = "Gateway.Actions.Exchange";
+  public static final String GATEWAY_ACTIONS_ROUTING_KEY = "Gateway.Action.Request";
+  public static final String GATEWAY_ACTIONS_DLQ = "Gateway.ActionsDLQ";
+  
+//===========================================================
+// //TODO rm adapter shouldnt have to declare the DLQ but if errors
```
Comment (garethTurner) [inline src/main/java/uk/gov/ons/fwmt/census/rmadapter/config/GatewayActionsQueueConfig.java]: fix TODOs

File: src/main/java/uk/gov/ons/fwmt/census/rmadapter/message/ActionInstructionReceiver.java [hunk 3]
```diff
@@ -13,30 +13,30 @@
...
 
   @Autowired
   private RMAdapterService rmAdapterService;
 
-  public void receiveMessage(String createJobRequestXML) throws CTPException {
+  public void receiveMessage(String message) throws GatewayException {
     try {
       //TODO Move this Queue Config
       //===================================================
       JAXBContext jaxbContext = JAXBContext.newInstance(ActionInstruction.class);
```
Comment (garethTurner) [inline src/main/java/uk/gov/ons/fwmt/census/rmadapter/message/ActionInstructionReceiver.java:30]: we should change this to reuse the same instance

**Review by garethTurner — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/ons/fwmt/census/rmadapter/canonical/CanonicalJobHelper.java (hunk 1 above)
Comment (sdiaz1000) [reply src/main/java/uk/gov/ons/fwmt/census/rmadapter/canonical/CanonicalJobHelper.java:37]: CaseRef is mapped to caseRef and CaseId is an additional property

**Review by garethTurner — APPROVED** [review]

**Pull request merged**